packages/cardpay-reward-programs/cardpay_reward_programs/main.py

```diff
@@ -13,34 +14,19 @@
 load_dotenv()
 
 cached_client = S3Client(
-    local_cache_dir="mycache",
+    local_cache_dir=".cache",
     boto3_session=Session(
-        aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
-        aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
     ),
 )
 cached_client.set_as_default_client()
 
 
-def select_rule(name, core_parameters, user_defined_parameters):
-    rule_constructor = globals()[name]
-    instance = rule_constructor(core_parameters, user_defined_parameters)
-    return instance
-
-
-def to_rule(parameters):
-    core_parameters, user_defined_parameters = get_parameters(parameters)
-    name = to_camel_case(core_parameters["docker_image"])
-    return select_rule(name, core_parameters, user_defined_parameters)
-
-
 def run_reward_program(
-    reward_program_id: str,
```

Comment:
Positional arguments here will let us define a launch process that's easier to manage across images. The rule name being first means I can set the entrypoint on the docker images I think so that we can build one per image.